src/main.js

```diff
@@ -325,8 +334,19 @@ const ack = {
 };
 
 client.on("message", function (topic, payload) {
-  const contents = JSON.parse(payload.toString());
-  console.log(isOwnPayload(contents) ? "🦝" : "🤖", topic, contents);
+  let contents = null;
+  try {
+    contents = JSON.parse(payload.toString());
+    console.log(isOwnPayload(contents) ? "🦝" : "🤖", topic, contents);
+  } catch {
+    // If we can't parse the payload, just use the raw string.
+    contents = payload.toString();
+    console.log(
+      isOwnPayload(contents) ? "🦝" : "🤖",
+      topic,
+      contents.substring(0, 100)
+    );
+  }
```

Comment:
This feels a bit nasty. Can we do it conditionally based on the topic? e.g a list of raw topics that we don't attempt to parse as json? Or maybe even topics that don't begin with `astoria/`

Reply:
I've pushed [6ae33d5](https://github.com/srobo/kit-ui/pull/231/commits/6ae33d594529ccb074cf75774742c6c8d251a7f8) to resolve this.